iamproperty/iamproperty.github.io
# Ds 71/toggle buttons

## Summary of Changes:
1. Document toggle buttons

## Rationale
Move out and separate the toggle buttons to its own page to highlight them.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /elements/form/toggle

## Ticket(s)
DS-72

## Checklist

## Notes
